cadquery/occ_impl/exporters/__init__.py

```diff
@@ -26,9 +27,10 @@ class ExportTypes:
     TJS = "TJS"
     DXF = "DXF"
     VRML = "VRML"
+    VTP = "VTP"
 
 
-ExportLiterals = Literal["STL", "STEP", "AMF", "SVG", "TJS", "DXF", "VRML"]
+ExportLiterals = Literal["STL", "STEP", "AMF", "SVG", "TJS", "DXF", "VRML", "VTP"]
```

Comment:
The import/export doc will need to be updated for the inclusion of VTP at some point.

Reply:
Adding a note in the docs